contrib/test_jenkins.sh

```diff
@@ -561,6 +561,67 @@ run_ucx_perftest() {
 	fi
 }
 
+start_daemon() {
+	daemon_exe="$1"
+
+	# Find daemon port
+	dmn_port="$server_port"
+	step_server_port
+
+	# Mandatory options to run the daemon
+	dmn_env="UCX_TCP_CM_REUSEADDR=y UCX_MAX_RNDV_LANES=1 UCX_RNDV_THRESH=0 UCX_RNDV_SCHEME=get_zcopy"
+
+	# Run the daemon
+	env $dmn_env $daemon_exe -p $dmn_port &
+
+	# Return the daemon pid and port
+	eval "$2=$!"
+	eval "$3=$dmn_port"
+}
+
+#
+# Run UCX performance daemon test
+#
+run_ucx_perftest_daemon() {
+	ucx_inst_ptest=$ucx_inst/share/ucx/perftest
+
+	ucx_perftest="$ucx_inst/bin/ucx_perftest"
+	ucx_perftest_daemon="$ucx_inst/bin/ucx_perftest_daemon"
+	ucp_test_args="-b $ucx_inst_ptest/test_types_ucp_daemon"
+
+	devices="$(get_active_ib_devices)"
+	devices="$(get_ib_bf_devices $devices)"
+
+	my_devices=$(get_my_tasks $devices)
+	for ucx_dev in $my_devices
+	do
+		echo "==== Running ucx_perftest_daemon kit on $ucx_dev ===="
+		export UCX_NET_DEVICES=$ucx_dev,lo
```

Comment:
why need lo ?

Reply:
Right, in principle that should work without loopback interface, we can just specify `0.0.0.0` address for daemons.
However, this didn't work out of the box. When host tries to connect to daemon, I see an error on daemon side:
```
    tcp_listener.c:136  UCX  DEBUG created a TCP listener 0x1cfaec0 on cm 0x1cd6400 with fd: 59 listening on 0.0.0.0:1338
   tcp_sockcm_ep.c:1128 UCX  DEBUG server created an endpoint on tcp_sockcm 0x1cd6400 id: 0 state: 1
           async.c:232  UCX  DEBUG added async handler 0x7ff8380011c0 [id=60 ref 1] uct_tcp_sa_data_handler() to hash
           async.c:494  UCX  DEBUG listening to async event fd 60 events 0x5 mode thread_spinlock
   tcp_sockcm_ep.c:360  UCX  DEBUG ep 0x7ff8380008c0 (fd=60 state=1): remote peer (127.0.0.1:44684) disconnected/rejected (Endpoint is not connected)
```
So on the daemon server side the remote peer (i.e. host client process) is still processed as `127.0.0.1:44684`.
Honestly I didn't spent much time debugging it, just enabled loopback interface to make it working.

Why 0.0.0.0 works for host-to-host communication?
Because there we don't use UCP EP, but plain TCP socket communication

Reply:
can we listen on the network address of $ucx_dev instead of 0.0.0.0 ?